Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,147 @@
-    "#### 5- Conclude the post-processing and validation with a last conclusive paragraph highlighting the pros and cons of the CFD simulations.\n",
-    "#### 6- Cite the source(s) that you have used for writing the content of your Domain file considering the copy right rules.  Make sure to cite the sources you use or refer to within the body of the text as well as in the end!\n",
-    "\n",
-    "# Handy Commands for Writing the Physics file: \n",
+    "<img src=\"./Images/residuals.png\" width = \"600\">\n",
+    "Fig. 1.\n",
     "\n",
-    "## Insert a Figure\n",
+    "The second step for post processing is to visualize the important flow field variables within the CFD domain. This visualization step can be problem specific and should be performed over an adequate plane of choice. For most of the problems contour of normalized velocity or pressure, over a horizontal or vertical plane at the center of the CFD domain, is a good start point. However, in this problem visualization of the vorticity field would be more beneficial to understand the physics of the flow and perform further investigation on it. The path to define an adequate visualization probe such as planes or lines is <span style=style=\"background-color:lightgrey;\">surface/plane-bounded</span>. With this path a user can define various type of surface probes to visualize the obtained numerical results over them. However, for the current problem since the entire domain is two dimensional and planar, it would make the entire domain the best candidate for the post-processing plane. User should now choose the variable of interest that needs to be visualized. The variable of interest here is the vorticity magnitude. It order to visualize the defined normalized flow field variable in the previous step users should use <span style=style=\"background-color:lightgrey;\">display/contour/vorticity-mag</span> and set the range of [-0.3,0.3] to visualize the vorticity contours and it's legend as shown in Fig. 2.\n",
     "\n",
-    "```\n",
-    "<img src=\"./Images/sample.jpg\" width=\"500\">\n",
-    "</br>\n",
-    "Fig. 1 - The caption for the image goes here.\n",
-    "```\n",
-    "\n",
-    "<img src=\"./Images/sample.jpg\" width=\"500\">\n",
-    "</br>\n",
-    "Fig. 1 - The caption for the image goes here."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Insert a YouTube video Available on the SFO Channel \n",
+    "<img src=\"./Images/vortices.png\" width = \"600\">\n",
+    "Fig. 2. Visualized vorticity field within the CFD domain for case studies with different rotational velocities (i.e. $\\alpha$).\n",
```

Comment:
Please note the style of writing the procedure to explain steps of post processing. Here the focus is to discuss general methodology for post process in parallel to discussing the physics.

I moved vorticity contours to the top section, because this provide a more clear picture of the flow field physics and develop an introduction in mind of the reader for next steps of post processing.